packages/stems/src/components/Popup/types.ts

```diff
@@ -52,7 +52,7 @@ export type PopupProps = {
    * to be the size of the container it belongs to. If the popup expands outside
    * the bounds of the container, it repositions itself.
    */
-  containerRef?: MutableRefObject<HTMLDivElement | undefined>
+  containerRef?: RefObject<HTMLElement>
```

Comment:
This one is a breaking change since it was `HTMLDivElement | undefined` before but is `HTMLDivElement | null` now

Reply:
i like the change to HtmlElement